pkgs/development/libraries/yajl/default.nix

```diff
@@ -1,18 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, cmake }:
+{ lib, stdenv, fetchFromGitHub, cmake, testers }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "yajl";
-  version = "2.1.0";
+  version = "unstable-2022-04-20";
 
   src = fetchFromGitHub {
-    owner = "lloyd";
+    owner = "containers";
     repo = "yajl";
-    rev = "refs/tags/${version}";
-    sha256 = "00yj06drb6izcxfxfqlhimlrb089kka0w0x8k27pyzyiq7qzcvml";
+    rev = "49923ccb2143e36850bcdeb781e2bcdf5ce22f15";
+    hash = "sha256-9bMPA5FpyBp8fvG/kkT/MnhYtdqg3QzOnmDFXKwJVW0=";
   };
 
+  patches = lib.optionals stdenv.hostPlatform.isStatic [
+    ./disable_shared.patch
```

Comment:
Can we not fix upstream to not generate shared libraries when BUILD_SHARED_LIBS is off?

Reply:
Considering the amount of open PRs including those about CVEs on the 
https://github.com/lloyd/yajl repository I doubt that would go anywhere, but the containers fork might work.

Although just turning off the shared libraries might not be enough, would a `-lyail` accept `libyail_s.a` as a valid link target?

Reply:
Turns out it was easier than I though: https://github.com/containers/yajl/pull/1

Will amend PR to fetch the patch from upstream.